command/daemon/flag/flag.go

```diff
@@ -0,0 +1,18 @@
+package flag
+
+import (
+	"github.com/giantswarm/microkit/command/daemon/flag/config"
+	"github.com/giantswarm/microkit/command/daemon/flag/server"
+	"github.com/giantswarm/microkit/flag"
+)
+
+type Flag struct {
+	Config config.Config
+	Server server.Server
+}
+
+func New() Flag {
+	f := Flag{}
+	flag.Init(&f)
```

Comment:
This is the trick of the flag initialization. An empty flag structure is provided to `flag.Init` and it applies the JSON path to the flag values according to their flag structure. See the package tests below for examples.